drivers/dai/intel/dmic/Kconfig.dmic

```diff
@@ -0,0 +1,76 @@
+# SOF SSP configuration options
```

Comment:
please use same structure used in other drivers, here I would put this file in drivers/dai/dmic/Kconfig.dmic_intel 
and the source/header files in drivers/dai/dmic/dmic_intel.c and drivers/dai/dmic/dmic_intel.h.

Reply:
Are you sure about dmic_intel?
In dai we have
alh - alh.c, Kconfig.alh
ssp - ssp.c Kconfig.ssp
dmic - dmic.c Kconfig.dmic

So only DMIC will be different with _intel

Changed "SOF SSP configuration" to "SOF DMIC configuration"

Reply:
I am looking now at drivers/dai and see the intel folder already exists in there, so this is following the same model. making  dmic different is weird now, we can move things around later if needed. good as for now.